web/packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -288,6 +288,9 @@ const EventIconMap: Record<EventCode, any> = {
   [eventCodes.INTEGRATION_CREATE]: Icons.Info,
   [eventCodes.INTEGRATION_UPDATE]: Icons.Info,
   [eventCodes.INTEGRATION_DELETE]: Icons.Info,
+  [eventCodes.STATIC_HOST_USER_CREATE]: Icons.Info,
```

Comment:
Should we use a user icon?

Reply:
lets do it (i'm not sure why we didn't in the first place 🤔 )